app/renderer/components/Hint.js

```diff
@@ -8,8 +8,8 @@ import { colors } from 'theme';
 const Wrapper = styled.div`
   margin-left: 0.5rem;
 
-  p + p {
-    margin-top: 1rem;
+  p {
+    max-width: 80vw;
```

Comment:
We have confirmed with the designer, the expected `max-width` is `25rem`.

We will include designers' expectation in further issues.

Please also put related issue id in PR title.

Otherwise, LGTM.

Reply:
Added #76 to PR title and commented on https://github.com/cennznet/runanode/issues/76.

Reply:
[ ] Could you please change `max-width` to `25rem`.